content/CORETutorials/Storage/Disks/DiskReplace.md

```diff
@@ -83,20 +86,68 @@ The new disk must have the same or greater capacity as the disk you are replacin
 The replacement fails when the chosen disk has partitions or data present.
 To destroy any data on the replacement disk and allow the replacement to continue, set the **Force** option.
 
-When the disk wipe completes and TrueNAS starts replacing the failed disk, the **Pool Status** changes to show the in-progress replacement.
+When the disk wipe completes and TrueNAS starts replacing the failed disk, the **Pool Status** screen changes to show the in-progress replacement.
 
 {{< trueimage src="/images/CORE/Storage/StoragePoolsStatusReplaceStart.png" alt="Replacing Started" id="Pool Status replacing disk" >}}
 
 TrueNAS resilvers the pool during the replacement process.
 For pools with large amounts of data, resilvering can take a long time.
 
-### Bringing a New Disk Online
-
-When the resilver completes, the pool status screen updates to show the new disk, and the pool status returns to **Online**.
+When the resilver completes, the **Pool Status** screen updates to show the new disk, and the pool status returns to **Online**.
 
 {{< trueimage src="/images/CORE/Storage/StoragePoolsStatusReplaceComplete.png" alt="Replacement Complete" id="Pool Status disk replacement complete" >}}
 
-During the failed disk replacement process, take these actions after removing and replacing the physical disk to make that replacement disk available:
-1. Go to **Disks** and locate the offline disk
-2. Click the <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> icon for the offline disk
-3. Click **Online**.
+## Replacing a Failed Disk with a Hot Spare
+
+A **Hot Spare** vdev sets up drives as reserved to prevent larger pool and data loss scenarios TrueNAS automatically inserts an available hot spare into a **Data** vdev when an active drive fails.
```

Comment:
This is awkwardly worded, I had to reread it a few times and it still didn't make total sense. Is this what you meant? 
```suggestion
A **Hot Spare** vdev sets up drives as reserved to prevent larger pool and data loss scenarios. TrueNAS automatically inserts an available hot spare into a **Data** vdev when an active drive fails.
```